apps/user/src/user.controller.ts

```diff
@@ -70,7 +70,7 @@ export class UserController {
 
   @MessagePattern({ cmd: 'get-user-by-mail' })
   async findUserByEmail(payload: { email }): Promise<object> {
-    return this.userService.findUserByEmail(payload);
```

Comment:
Please check if this function `findUserByEmail` is used somewhere before removing it.

Reply:
Resolved.